### PR TITLE
missing bosh completion code

### PR DESCRIPTION
**Pull request by kinjelom**
The missing piece of code making sure that completion works not only in tests ;)
Related to:
- https://github.com/cloudfoundry/bosh-cli/pull/645
- https://github.com/cloudfoundry/bosh-cli/pull/646

### CODE REVIEW

**Comment by jpalermo** [comment]
Replacement PR for #646 which was closed. Adding back the same reviewers.

File: main.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/cloudfoundry/bosh-cli/v7/cmd/completion"
```
Comment (aramprice) [inline main.go:5]: this should move down around like 16 / 17

**Review by aramprice — APPROVED** [review]
lgtm, thanks!

**Review by rkoster — APPROVED** [review]
Works on my machine :)

**Comment by rkoster** [comment]
Thanks! @kinjelom

**Pull request merged**